src/services/io/podio/JEventSourcePODIO.cc

```diff
@@ -267,7 +267,7 @@ double JEventSourceGeneratorT<JEventSourcePODIO>::CheckOpenable(std::string reso
 
     // PODIO FrameReader segfaults on legacy input files, so we use ROOT to validate beforehand. Of course,
     // we can't validate if ROOT can't read the file.
-    std::unique_ptr<TFile> file = std::make_unique<TFile>(resource_name.c_str());
+    std::unique_ptr<TFile> file = std::unique_ptr<TFile>{TFile::Open(resource_name.c_str())};
```

Comment:
I don't see how TFile::Open will be different here. From what I understand this function isn't meant to check if file exists, but check it it's a compatible format. There is also a legacy PODIO version of the source that can be chosen instead.

Reply:
There is a very useful error (and JANA) from root before making this change:
```
Error in <TFile::TFile>: please use TFile::Open to access remote files:
        auto f = std::unique_ptr<TFile>{TFile::Open("root://dtn-eic.jlab.org//work/eic2/EPIC/RECO/24.08.1/epic_craterlake/SIDIS/pythia6-eic/1.0.0/10x100/q2_0to1/pythia_ep_noradcor_10x100_q2_0.000000001_1.0_run9.ab.0544.eicrecon.tree.edm4eic.root")};
----------------------------------------------------------
JException
  Message:  Unable to open event source "root://dtn-eic.jlab.org//work/eic2/EPIC/RECO/24.08.1/epic_craterlake/SIDIS/pythia6-eic/1.0.0/10x100/q2_0to1/pythia_ep_noradcor_10x100_q2_0.000000001_1.0_run9.ab.0544.eicrecon.tree.edm4eic.root": No suitable generator found!
```